Runtime/Scripts/PersistentConnection.cs

```diff
@@ -241,14 +260,19 @@ void StartEdgeEventsLocation(EdgeEventsConnection connection)
 
     void StartEdgeEventsLatency(EdgeEventsConnection connection, string host)
     {
+      Logger.Log("EdgeEvents Posting latency update, Location to send [" + location.Latitude + ", " + location.Longitude + "]");
+      bool requestSent;
       if (hasTCPPorts)
       {
-        connection.TestConnectAndPostLatencyUpdate(host, (uint)config.latencyTestPort, location).ConfigureAwait(false);
+        requestSent = connection.TestConnectAndPostLatencyUpdate(host, (uint)config.latencyTestPort, location).ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
Or skip await since it's mostly intended to be fire and forget? (or use await <task>.ConfigureAwait(false) to actually wait on current thread for the background task).

Reply:
As discussed.